modules/nf-core/wfmash/main.nf

```diff
@@ -0,0 +1,44 @@
+process WFMASH {
+    tag "$meta.id"
+    label 'process_medium'
+
+    conda "bioconda::wfmash=0.10.4"
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/wfmash:0.10.4--hea8008d_0':
+        'biocontainers/wfmash:0.10.4--hea8008d_0' }"
+
+    input:
+    tuple val(meta), path(fasta_gz), path(paf), path(gzi), path(fai)
+    val(query_self)
+    path(fasta_query_list)
+
+    output:
+    tuple val(meta), path("*.paf"), emit: paf
+    path "versions.yml"           , emit: versions
+
+
+    when:
+    task.ext.when == null || task.ext.when
+
+    script:
+    def args = task.ext.args ?: ''
+    def prefix = task.ext.prefix ? task.ext.prefix : paf ? "${meta.id}" + "." + paf.baseName.split("\\.")[-1] : "${meta.id}"
+    def query_list = fasta_query_list ? "--query-file-list ${fasta_query_list}" : ""
+    def query = query_self ? "${fasta_gz}" : ""
+    def paf_mappings = paf ? "--input-paf ${paf}" : ""
+    """
+    wfmash \\
+        ${fasta_gz} \\
+        $query \\
+        $query_list \\
+        --threads $task.cpus \\
+        $paf_mappings \\
+        $args > ${prefix}.paf
```

Comment:
On start, wfmash options do not seem to have defaults

Wfmash Options
  wfmash_block_length       : null
  wfmash_sparse_map         : null

shows up as

Wfmash Options
wfmash_block_length  N/A
wfmash_sparse_map  N/A

in the MultiQC report, under nf-core/pangenome Workflow Summary

Reply:
I was also seeing this and annoyed.
These parameters can be set, but if they are not set, they will be calculated based on other parameters later. So I am not sure what to do here.

Reply:
Because one should never set `params.<PARAM>` later.

Reply:
I see, haven't run into this before.  Maybe drop a question on nf-core slack to see if anyone else might have a suggestion.

Reply:
https://nfcore.slack.com/archives/CE6SDBX2A/p1689605084916579

I tried, but no answer, yet.